runtime/src/iree-amd-aie/driver/hsa/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc. All Rights Reserved.
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+iree_add_all_subdirs()
+
+iree_register_external_hal_driver(
+  NAME
+    amd-aie-hsa
+  DRIVER_TARGET
+    iree-amd-aie::driver::hsa::registration
+  REGISTER_FN
+    iree_hal_hsa_driver_module_register
+)
+
+include(iree_aie_utils)
+replace_string_in_file(
+  "${ROCR_RUNTIME_ROOT}/runtime/hsa-runtime/core/inc/amd_gpu_agent.h"
+  "static const" "const")
```

Comment:
This can be put in your branch?

Reply:
The review process for stuff over there is much different from ours (the actual source repo is on gerrit...) so I'm not ready to start making/proposing changes to ROCr outside of the AIE specific files. But actually this sed can be removed now that I'm not supporting a "direct link" mode.